docker-envs/base-ror-docker/es/install-ror-es-using-file.sh

```diff
@@ -13,6 +13,6 @@ echo "Installing ES ROR from file..."
 /usr/share/elasticsearch/bin/elasticsearch-plugin install --batch file:///tmp/ror.zip
 if verlte "6.5.0" "$ES_VERSION"; then
   echo "Patching ES ROR..."
-  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch --I_UNDERSTAND_AND_ACCEPT_ES_PATCHING yes
+  /usr/share/elasticsearch/jdk/bin/java -jar /usr/share/elasticsearch/plugins/readonlyrest/ror-tools.jar patch --I_UNDERSTAND_AND_ACCEPT_ES_PATCHING=yes
```

Comment:
hmmm, I've just noticed this subtle difference. Space vs `=`. 
Why did it change?

Reply:
I changed it to be consistent with KBN tools - they use the syntax with `=` everywhere. Both syntaxes are supported and I added tests - both variants are tested.